layers/meta-resin-genericx86/recipes-kernel/linux/files/realsense_metadata_linux-yocto_4.4.patch

```diff
@@ -0,0 +1,210 @@
+From c5811c81372e9fe5b5216074a194c1be1ff12d1f Mon Sep 17 00:00:00 2001
+From: Scott Ware <scott.r.ware@intel.com>
+Date: Mon, 27 Nov 2017 15:26:49 +0000
+Subject: [PATCH 3/4] Realsense metadata
+
+Upstream-status: Backport
```

Comment:
needs also signed-ff by you, Theodor, in the correct order.
See https://github.com/resin-os/resin-intel/blob/master/layers/meta-resin-genericx86/recipes-kernel/linux/files/0001-Add-support-for-Quectel-EC20-modem.patch for example